README.md

```diff
@@ -327,54 +327,56 @@ frontend and backend image resource to the correct url with your namespace name
 
 
 A `PipelineRun` is how you can start a pipeline and tie it to the persistentVolumeClaim and params that should be used for this specific invocation.
-
-Lets start a pipeline to build and deploy backend application using `tkn`:
+ 
+Let's start a pipeline to build and deploy the backend application using `tkn`:
 
 ```bash
 $ tkn pipeline start build-and-deploy \
+    --prefix-name build-deploy-api-pipelinerun \
     -w name=shared-workspace,volumeClaimTemplateFile=https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/01_pipeline/03_persistent_volume_claim.yaml \
     -p deployment-name=pipelines-vote-api \
     -p git-url=https://github.com/openshift/pipelines-vote-api.git \
     -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/pipelines-vote-api \
     --use-param-defaults
 
 
-Pipelinerun started: build-and-deploy-run-z2rz8
+Pipelinerun started: vote-api-t9tpq
```

Comment:
similarly here why this change?

Reply:
have a more meaningful name for the run. Better than the generic one. Otherwise you don't know which object is built and deployed.